src/main/resources/templates/snowflake/Makefile-configDiff.schema.ssp

```diff
@@ -0,0 +1,12 @@
+<%@ val tableDiffs: Seq[io.phdata.streamliner.schemadefiner.model.TableDiff] %>
```

Comment:
Not sure if first-run-all, run-all and clean-all all three needs to be added. 
I have added  first-run-all for now.

Reply:
Ideally we just have "run" and "run-all". If first-run is required that is ok.